Apollo/Components/FadeThumb.xaml

```diff
@@ -7,6 +7,20 @@
   <UserControl.Resources>
     <SolidColorBrush x:Key="Outline">#00FFFFFF</SolidColorBrush>
     <SolidColorBrush x:Key="Color">#FFFFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="ContextItemSelected">#119EDA</SolidColorBrush>
+
+    <Ellipse x:Key="SelectedIcon" Margin="7 0 0 0" Fill="{DynamicResource ContextItemSelected}" Height="10" Width="10" HorizontalAlignment="Center" VerticalAlignment="Center"/>
```

Comment:
There is an AccentColor resource available for the Fill color (you can find the brush in the Themes folder)